### DISCUSSION TITLE
Is there a third-party library available to turn on and off full screen mode

### DISCUSSION THREAD

**Discussion by ShaoGongBra** [microsoft/react-native-windows#12483] [Q&A] [answered]
Using code to control entering and exiting full screen

**Comment by ghost** [comment, marked as answer]
[react-native-custom-window](https://github.com/Victor0814gui/react-native-custom-window) still needs better documentation and publishing, but the function ```window.fullscreen()``` does that. I still need to improve some things, but it works well; I would appreciate feedback to enhance it.

**Reply by ShaoGongBra** [reply]
Thank you for testing it

**Reply by ShaoGongBra** [reply]
An unexpected error occurred: "https://registry.npmjs.org/react-native-custom-window: Not found"

**Reply by ShaoGongBra** [reply]
I successfully installed from GitHub

**Reply by ghost** [reply]
I haven't published it yet😥; I set the project aside for a while, but I'll publish it as urgently as possible😁.

**Reply by ShaoGongBra** [reply]
The full screen method does not work in Release mode

**Reply by ghost** [reply]
I did some tests here, however, I had the same error. I will check more carefully.

**Reply by ShaoGongBra** [reply]
May I ask if there is any progress

**Comment by ghost** [comment]
Fixed, what happened was that in the release, the nativeModules in some implementations were not available, but I managed to solve it by replacing them with turboModules.
